tests/acceptance/index-test.js

```diff
@@ -0,0 +1,149 @@
+import { click, currentURL, fillIn, findAll, visit } from '@ember/test-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { setupApplicationTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+import setupPullRequestsTest from 'whats-new-in-emberland/tests/helpers/pull-requests';
+
+
+module('Acceptance | index', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupPullRequestsTest(hooks);
```

Comment:
Calling `setupPullRequestsTest` makes the custom assertion `isPullRequestCorrect` available to each test.